images/router/haproxy/conf/haproxy-config.template

```diff
@@ -206,18 +206,25 @@ backend openshift_default
             where to send the traffic but should run the be in tcp mode
         3. if the config is terminated at the
 */}}
-{{ range $id, $serviceUnit := .State }}
-        {{ range $cfgIdx, $cfg := $serviceUnit.ServiceAliasConfigs }}
-            {{ if or (eq $cfg.TLSTermination "") (eq $cfg.TLSTermination "edge") }}
-                {{ if (eq $cfg.TLSTermination "") }}
+{{ range $cfgIdx, $cfg := .State }}
+  {{ if or (eq $cfg.TLSTermination "") (eq $cfg.TLSTermination "edge") }}
+    {{ if (eq $cfg.TLSTermination "") }}
+# Plain http backend
 backend be_http_{{$cfgIdx}}
-                {{ else }}
+    {{ else }}
+# Plain http backend but request is TLS, terminated at edge
 backend be_edge_http_{{$cfgIdx}}
-                {{ end }}
+    {{ end }}
   mode http
   option redispatch
   option forwardfor
+    {{ with $balanceAlgo := index $cfg.Annotations "haproxy.balance" }}
```

Comment:
Namespace the annotation just like other existing k8s/openshift annotations, router.openshift.io/haproxy.balance?